### ISSUE TITLE
Set custom icon for notification

### ISSUE THREAD

**Issue by farhan687** [intercom/intercom-cordova#210]
Build failed while I try to set the custom icon for notification.

## Version info
  - Cordova version:6.2.3
  - Intercom cordova version: 3.2.2
  - Android OS version: 6.0.1
  - iOS/Android Device: CPH1701

## Expected behavior
Notification icon should be set.

## Actual behavior
Build failed

## Steps to reproduce
 1. Set custom icons using following naming conventions

/res/drawable-xxhdpi/intercom_push_icon.png 
/res/drawable-xhdpi/intercom_push_icon.png 
/res/drawable-hdpi/intercom_push_icon.png 
/res/drawable-mdpi/intercom_push_icon.png

## Logs
```
Error: /app/platforms/android/gradlew: Command failed with exit code 1 Error output:
/app/platforms/android/build/intermediates/res/merged/debug/drawable-hdpi-v4/intercom_push_icon.png: error: Duplicate file.
/app/platforms/android/build/intermediates/res/merged/debug/drawable-hdpi/intercom_push_icon.png: Original is here. The version qualifier may be implied.


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

```

**Comment by ronocod** [comment]
Can you update to version 4.1.0 of this plugin and 6.3.0 of Cordova Android and try again?

**Comment by bboyle18** [comment]
Should be resolved by updating to v4.1.0 of the Intercom Cordova plugin.
Closing for now. Feel free to reopen if you are still experiencing this issue.
